examples/lib/stories/bridge_libraries/flame_isolate/simple_isolate_example.dart

```diff
@@ -2,12 +2,13 @@ import 'dart:math';
 
 import 'package:flame/components.dart';
 import 'package:flame/effects.dart';
+import 'package:flame/experimental.dart';
```

Comment:
why is this needed here?

Reply:
Because the `TapCallbacks` mixin is exposed from there, might be time to expose it elsewhere too.

Reply:
I would say so, yeah, since we are making it general available, and the older API is now deprecated, I would argue that this is not experimental anymore.

Reply:
The older API is not deprecated yet, it is only the `HasTappableComponents` mixin that is deprecated in this PR, and that mixin is part of the new API. I would keep it as experimental for one more version to make sure that we don't want to make any more breaking changes to it.

Reply:
Got ya, so the older api can be used yet, you just don't need the mixin on the game class, sounds good then!!!

Reply:
You almost got it, the older API is using a mixin called `HasTappables` and that one has to still be used if one wants to use the `Tappable` API instead of the `TapCallbacks` API.